test/files/neg/t2102.check

```diff
@@ -1,6 +1,6 @@
 t2102.scala:2: error: type mismatch;
- found   : java.util.Iterator[Int]
- required: scala.collection.Iterator[_]
+ found   : Iterator[Int] (in java.util)
+ required: Iterator[_] (in scala.collection)
```

Comment:
Firstly: very cool idea!
Could we also align the `(` segments to easier visually identify where the mismatch is? E.g. what I often find:

```scala
Thing[Bla]    (in akka.stream.scaladsl)
Thing[Blabla] (in akka.stream.javadsl)
```

Which makes it even easier to spot "aha! the diff is the javadsl/scaladsl" and not that the type parameter was the problem (it could be fine, e.g. because Bla extended Blabla etc).